omod/src/main/java/org/openmrs/module/reporting/web/validator/RunReportFormValidator.java

```diff
@@ -0,0 +1,89 @@
+package org.openmrs.module.reporting.web.validator;
+
+import org.openmrs.module.reporting.common.ObjectUtil;
+import org.openmrs.module.reporting.evaluation.EvaluationUtil;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.reporting.web.reports.CommandObject;
+import org.quartz.CronExpression;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+import java.util.*;
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
@dkayiwa This class was removed as the validator logic added back to the controller